spec/controllers/fee_categories_controller_spec.rb

```diff
@@ -25,7 +25,10 @@
       before do
         ['General Applications',
          'Money Claims'].each do |category|
-          FeeCategory.create!(title: category, statutory_instrument_id: 1)
+          FeeCategory.create!(title: category,
+                              statutory_instrument_id: 1,
+                              description: 'description',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.